pkg/apis/elasticsearch/v1alpha1/types.go

```diff
@@ -85,11 +85,8 @@ type ElasticsearchNodeSpec struct {
 }
 
 // ElasticsearchSecure struct represents security configuration of the cluster
-// whether SearchGuard is enabled along with oauth-proxy sidecar
 type ElasticsearchSecure struct {
-	Disabled           bool   `json:"disabled"`
-	Image              string `json:"image,omitempty"`
-	CertificatesSecret string `json:"certificatesSecret,omitempty"`
+	CertificatesSecret string `json:"certificatesSecret"`
```

Comment:
I dont think it necessary to keep this nested under `ElasticsearchSecure` if its the only element here.